### DISCUSSION TITLE
Duplicated feeds

### DISCUSSION THREAD

**Discussion by Nikodemsky** [martinrotter/rssguard#705] [Q&A]
So i have this feed:
https://politepol.com/fd/yiLx1BSALEq0
- it updates itself every hour with any new posts from current day.

Thing is, that rssguard is downloading ALL the messages at every interval, not only "new" feeds(i'm always deleting articles that i have read).

I guess that problem is with the feed itself(since i have no problem with other feeds), but i'm not sure what it is, can anyone pinpoint me how this feed needs to be formatted, so the app won't download same articles over and over again?

**Comment by martinrotter** [comment]
Will check.

**Comment by martinrotter** [comment]
I found a problem with the feed.

The feed always updates its "description" tag with polish (?) texts like:
* około 13 godzin temu
* około 10 godzin temu
* etc

These text change each hour. I suspect they are the problem. I will double check and report back.
